scarb/src/compiler/compilers/starknet_contract.rs

```diff
@@ -5,7 +5,7 @@ use anyhow::{bail, Context, Result};
 use cairo_lang_compiler::db::RootDatabase;
 use cairo_lang_starknet::allowed_libfuncs::{
     validate_compatible_sierra_version, AllowedLibfuncsError, ListSelector,
-    DEFAULT_EXPERIMENTAL_LIBFUNCS_LIST,
+    BUILTIN_EXPERIMENTAL_LIBFUNCS_LIST,
```

Comment:
I'm assuming this is the same value, just renamed in the cairo2?

Reply:
It used to be "experimental_v0.1.0", now it is just "experimental", the underlying list has been renamed.